tests/hazmat/primitives/test_aes.py

```diff
@@ -60,6 +60,20 @@ def test_xts_too_short(self):
         with pytest.raises(ValueError):
             enc.update(b"0" * 15)
 
+    @pytest.mark.supported(
+        only_if=lambda backend:
+        (
+            backend._lib.CRYPTOGRAPHY_OPENSSL_111D_OR_GREATER,
+        ),
```

Comment:
This check isn't correct: You're returning a tuple from the callback, which always evaluates to truthy.

Reply:
derp, thanks!

Reply:
I've kept the parens, which were added to keep line length under flake8 limit - but removed the offending comma.

Reply:
You can (and likely will need to, for CI to pass) run `black`, which will deal with all these formatting issues.

Reply:
Seems I also missed that the `only_if` is only evaluated is only evaluated if the test function has a `backend` argument.

Reply:
Yeah, that's a very non-obvious requirement so not a surprise you missed it initially.